portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,25 @@
   </head>
   <body>
     <div class="card">
-      <h1>Connor's Portfolio</h1>
-      <img class= "avatar" src="https://i.imgur.com/AW18kUT.png" alt="avatar"/>
-      <p>My name is Connor and I am a rising junior at University of Arizona studying Computer Science.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Connor's Portfolio</h1>
```

Comment:
Let's follow https://google.github.io/styleguide/htmlcssguide.html#Indentation

Indent by 2 spaces at a time.

Reply:
I think the cloud shell editor defaults to 4, is there any way to change that?